docs/standardisation-scope.adoc

```diff
@@ -0,0 +1,199 @@
+= Standardisation scope of Cypher
+
+One of the challenges faced by this effort to standardise Cypher has been to choose the set of language features to standardise.
+Cypher is not a new language, but has several years of history, which includes some historical artifacts left in the feature set.
+This is generally not because there hasn't been good replacements, but because Cypher's role has been to support a specific product (Neo4j).
+The lifecycle of a standardised language is different, however, and removal of standardised features is generally hard.
+
+With that in mind, the openCypher project is trying to not include language features that are not visioned to be the future of the language.
```

Comment:
"is trying to not include language features that are not visioned to be the future of the language."
-->
"is endeavouring not to include language features that are not envisioned to be the future of the language."